src/tunnel.rs

```diff
@@ -413,31 +408,34 @@ make_tunnel_type! {
     NgrokLabeledTunnel, LabeledTunnel, label
 }
 
-pub async fn forward_tcp(id: &String, addr: String) -> PyResult<()> {
-    info!("Tunnel {id:?} TCP forwarding to {addr:?}");
-    let res = get_storage_by_id(id)
-        .await?
-        .tunnel
-        .lock()
-        .await
-        .fwd_tcp(addr)
-        .await;
-
-    debug!("forward_tcp returning");
-    canceled_is_ok(res)
-}
-
-pub async fn forward_pipe(id: &String, addr: String) -> PyResult<()> {
-    info!("Tunnel {id:?} Pipe forwarding to {addr:?}");
-    let res = get_storage_by_id(id)
-        .await?
-        .tunnel
-        .lock()
-        .await
-        .fwd_pipe(addr)
-        .await;
+pub async fn forward(id: &String, addr: String) -> PyResult<()> {
+    let tun = &get_storage_by_id(id).await?.tunnel;
+    let is_pipe =
+        addr.starts_with(PIPE_PREFIX) || addr.starts_with("unix:") || addr.starts_with("tun-");
```

Comment:
The `tun-` is just a convention in the test cases, probably doesn't make sense to bake in here. I do think we should have a `/` to cover fully-qualified unix paths. In the case where a user just has a filename it's hard to differentiate, is `my-project.so` a file or a domain from Somalia?

The most common case are domains, and the user has an out in that they can add `unix:` to force it if we don't auto-detect it happily, so this probably biases in the right direction. Are there other cases we could know it's a path, maybe if there's a forward-slash anywhere in the string? That would cover cases like `./my-project.sock`. They might have weird dns setup so can't rely on "must have a real internet TLD", wouldn't be fun to keep those up to date anyway.

In the end I'd recommend:
1. Automatically assume pipe if the string contains a `/` anywhere (if at the end it won't work as a domain anyway)
2. Remove the notion of `tun-` here. Will have to add `unix:` to the tests
3. Add a note in the comment about `unix:` being a way to force pipe behavior. Maybe that should go in the README too if there's a convenient spot for it.